### ISSUE TITLE
Dashboard polish

### ISSUE THREAD

**Issue by gibbsmaya** [OpenTechFund/opentech.fund#988]
Front end review on dashboard. Currently can see we need:

 - [x] some padding between blocks 
 - [x] Alignment of Table heading label when filters and search present (it appears higher up, should be inline with search and filters). note this is relevant for both the dashboard and tables on Submission overview page.
 - [x] Revisit the spacing in the table for submissions since we have no >.

**Comment by chris-lawton** [comment]
Ready for review - https://github.com/OpenTechFund/opentech.fund/pull/1015

Note that the first AC, padding between blocks, relies on https://github.com/OpenTechFund/opentech.fund/issues/959 so will create a separate PR for that.

**Comment by helenb** [comment]
Just one query in the code review.

**Comment by chris-lawton** [comment]
Hi Maya,

The second and third AC are now on dev but waiting on #959 in order to do the first one. 

Once you reviewed pass back it back to me and i'll create another PR for it. 

Thanks,
Chris

**Comment by gibbsmaya** [comment]
Hi @chris-lawton,

Thanks. Have reviewed and 3rd AC looks fine. Re second one, can the title of the block be in line with the search/filters as per the [design](https://projects.invisionapp.com/share/KZPIO1NENQJ#/screens/335805633)? Or is there a reason this is not possible?

Thanks,
Maya

**Comment by chris-lawton** [comment]
Hi @gibbsmaya, 

That can be done, I think i just interpreted the design so that heading and the search/filters and vertically centred.   

So to to confirm it should look like:
![screen shot 2019-02-14 at 15 59 29](https://user-images.githubusercontent.com/31627284/52799551-f583bd00-3071-11e9-8d1f-687531be3a0f.png)

and not:
![screen shot 2019-02-14 at 15 59 49](https://user-images.githubusercontent.com/31627284/52799556-fa487100-3071-11e9-93f7-a6142c21d3f1.png)

Thanks,
Chris

**Comment by chris-lawton** [comment]
Hi Maya, 

The first and second AC are now live on dev. 

Thanks,
Chris

**Comment by gibbsmaya** [comment]
LGTM. Handing over to @todd-dembrey for deployment to test.
